run.sh

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env bash
+
+if [ $# -eq 1 ] && ([ "$1" == "bash" ] || [ "$1" == "cat" ]); then
```

Comment:
I added `cat` here to 'pause' the container on startup allowing for `exec` testing